Sources/JavaKit/JavaKitVM/JavaVirtualMachine.swift

```diff
@@ -268,4 +280,8 @@ extension JavaVirtualMachine {
       }
     }
   }
+
+  enum JavaKitError: Error {
+    case classPathNotFound(String)
```

Comment:
```suggestion
    case classPathSegmentNotFound(notFound: String, classpath: String)
```

Reply:
Yes, let's just check `segment` is the right word, I'll look at the JDK source to see if it suggests anything (unless you know this for sure?)

Reply:
I think `entry` is used in the JDK.

Reply:
Good catch, sounds good!

Reply:
And just to confirm you don't want to make `VMError` a case of `JavaKitError`?

Reply:
Either is fine tbh, whichever you prefer. I don’t think we expect people to handle them specifically